app/package.json

```diff
@@ -75,37 +76,36 @@
     "prettier": "^3.3.3",
     "relay-compiler": "^16.2.0",
     "rimraf": "^5.0.9",
+    "rollup-plugin-visualizer": "^5.12.0",
     "ts-jest": "^29.2.2",
-    "typescript": "~5.4.5"
+    "typescript": "~5.4.5",
+    "vite": "^5.3.3",
+    "vite-plugin-relay": "^2.1.0"
   },
   "scripts": {
     "preinstall": "npx only-allow pnpm",
-    "clean": "rimraf ../src/phoenix/server/static",
-    "build": "pnpm run clean && pnpm run build:static & pnpm run build:relay && pnpm run build:ts",
-    "build:static": "cpy ./static ../src/phoenix/server",
-    "build:ts": "node ./esbuild.config.mjs",
+    "build": "pnpm run build:relay && vite build",
     "build:relay": "relay-compiler",
-    "watch": "./esbuild.config.mjs dev",
     "test": "jest --config ./jest.config.js",
-    "dev": "pnpm run dev:server:demo & pnpm run build:static && pnpm run watch",
+    "dev": " pnpm run dev:server:demo & pnpm run build:relay && vite",
```

Comment:
```suggestion
    "dev": "pnpm run dev:server:demo && vite",
```

Reply:
I don't think I fully follow but we can leave it for now. Can you remove the whitespace?